pages/web-literacy.jsx

```diff
@@ -49,7 +49,7 @@ var ActivitiesPage = React.createClass({
         <section>
           <WebMaps>
             <WebMap head="Explore" subhead="Reading the Web">
-              <Expander head="navigation">
+              <Expander head="navigation" anchorId="navigation">
```

Comment:
Note that `anchorId` is like a slug-ified version of `head`, so it seems like we might be able to determine it algorithmically to keep things DRY, but [it's a trap](https://www.youtube.com/watch?v=4F4qzPbcFiA) because we will eventually need to localize `head`, while we don't need to localize `anchorId`.